sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/AdtQueryBuilder.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text;
+using Azure.DigitalTwins.Core.QueryBuilder;
+
+namespace Azure.DigitalTwins.Core.Queries.QueryBuilders
+{
+    /// <summary>
+    /// Used to instantiate an ADT query.
+    /// </summary>
+    public class AdtQueryBuilder
+    {
+        private readonly SelectQuery _selectQuery;
+        private readonly FromQuery _fromQuery;
+        private readonly WhereQuery _whereQuery;
+
+        /// <summary>
+        /// Initializes the clauses that make up a standard query.
+        /// </summary>
+        public AdtQueryBuilder()
+        {
+            _whereQuery = new WhereQuery(this);
+            _fromQuery = new FromQuery(this, _whereQuery);
+            _selectQuery = new SelectQuery(this, _fromQuery);
+        }
+
+        /// <summary>
+        /// Used to select one or multiple collections or properties.
```

Comment:
I don't think we select collections do we? 
How about: 
`It specifies the list of columns that the query will return`
I got this straight from the ADT documentation page.